src/test/components/__snapshots__/Details.test.js.snap

```diff
@@ -475,9 +475,9 @@ exports[`app/Details show the correct state for the sidebar open button 2`] = `
   <TabBar
     extraElements={
       <button
-        className="sidebar-open-close-button photon-button photon-button-ghost sidebar-open-close-button-isclosed"
+        className="sidebar-open-close-button photon-button photon-button-ghost sidebar-open-close-button-isopen"
         onClick={[Function]}
-        title="Open the sidebar"
+        title="Close the sidebar"
```

Comment:
AH the new snapshots are actually what's expected -- this means the tests weren't working properly before :/

Reply:
This is https://github.com/airbnb/enzyme/pull/1499 -- and this is a good thing !

Before this we had to call "update" everywhere (I just forgot to do it here). I'm gonna remove them all then !

Reply:
Woops.